Sitemap/DepthGuesser.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+use Doctrine\Bundle\PHPCRBundle\ManagerRegistry;
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Symfony\Cmf\Bundle\SeoBundle\Model\UrlInformation;
+
+/**
+ * This guesser will add the depth of a document persisted on a phpcr node.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class DepthGuesser implements GuesserInterface
+{
+    /**
+     * @var ManagerRegistry
+     */
+    private $managerRegistry;
+
+    /**
+     * @var int The depth of the content base path as the depth offset.
+     */
+    private $offset;
+
+    /**
+     * DepthGuesser constructor.
+     *
+     * @param ManagerRegistry $managerRegistry
+     */
+    public function __construct(ManagerRegistry $managerRegistry, $contentBasePath)
+    {
+        $this->managerRegistry = $managerRegistry;
+        $this->offset = substr_count($contentBasePath, '/') + 1;
```

Comment:
i am not sure if + 1 is correct.

/ => 0
/cms => 1
/cms/content => 2

or am i miscounting here? i would expect the PHPCR depth to count from 0 as well. if base is /cms/content and i look at the depth of /cms/content, it would be 2, minus the 2 slashes i end up with 0.

we should handle the special case `'/' === $contentBasePath` btw.